src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1667,6 +1723,190 @@ int JasmineGraphFrontEnd::getRunningHighPriorityTaskCount() {
     return taskCount;
 }
 
+void JasmineGraphServer::pageRank(std::string graphID, double alpha) {
+
+    std::map<std::string, JasmineGraphServer::workerPartition> graphPartitionedHosts =
+            JasmineGraphServer::getWorkerPartitions(graphID);
+    int partition_count = 0;
+    string partition;
+    string host;
+    int port;
+    int dataPort;
+    std::string workerList;
+    Utils utils;
+
+    std::map<std::string, JasmineGraphServer::workerPartition>::iterator workerit;
+    for (workerit = graphPartitionedHosts.begin(); workerit != graphPartitionedHosts.end(); workerit++) {
```

Comment:
Better use the variable name `workerIter` than `workerit` here.

Reply:
changed

Reply:
Ok